package.json

```diff
@@ -30,5 +31,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "json-server": "^0.16.1",
+    "prettier": "^1.19.1",
+    "react-router-dom": "^5.1.2"
```

Comment:
react-router-dom has to be installed as a "normal" dependency not as devDependency